client/blocks/comments/post-comment-list.jsx

```diff
@@ -88,6 +88,7 @@ class PostCommentList extends React.Component {
 			commentsTree={ this.props.commentsTree }
 			commentId={ commentId }
 			key={ commentId }
+			showModerationTools={ this.props.showModerationTools }
```

Comment:
this is a better place to put the `config.isEnabled( 'moderation-tools-in-posts ')`